frontend/src/components/common/FileUpload.js

```diff
@@ -14,11 +14,13 @@ const propTypes = {
   uploadUrl: PropTypes.string.isRequired,
   maxFileSize: PropTypes.string,
   acceptedFileTypesMap: PropTypes.objectOf(PropTypes.string),
+  onFileLoad: PropTypes.func,
 };
 
 const defaultProps = {
   maxFileSize: "100MB",
   acceptedFileTypesMap: {},
+  onFileLoad: null
```

Comment:
ah, yes, this is exactly what should be used above, Except functions cannot be null, user `() => {}` instead.

Reply:
After discussion, added a comment explaining that FilePond wants this to be null.